environment-dev.yml

```diff
@@ -36,6 +36,7 @@ dependencies:
   - cloudpickle
   - boto3
   - scikit-learn
+  - sentinels
```

Comment:
We usually use `pandas._lib.no_default` for the same purpose, I think we can do the same here as well, instead of adding new dependencies
https://github.com/modin-project/modin/blob/34ee054bb623db3e77722f0fb69e16d0084e1e6c/modin/pandas/groupby.py#L21

Reply:
Done.